typescript/cli/src/config/warp.ts

```diff
@@ -180,7 +176,6 @@ export async function createWarpRouteDeployConfig({
           type,
           owner,
           proxyAdmin,
-          isNft,
```

Comment:
to ensure we not break anything, can you just set `isNft: true` for collateralUri and syntheticUri?

Reply:
Done